formicarium/test/UnitTest/diff_robot_unit_test.py

```diff
@@ -0,0 +1,144 @@
+from unittest import TestCase
+from ..Stub.Lidar_Stub import Lidar_Stub
+from formicarium import DiffRobot
+from ..Stub.Publisher_Stub import Publisher_Stub
+from pygame import Surface
+
+
+class DiffRobotUnitTest(TestCase):
+    def test_create_robot_nothrow(self):
+
+        lidar = Lidar_Stub()
+        odomPublisher = Publisher_Stub()
+        posePublisher = Publisher_Stub()
+
+        try:
+            robot = DiffRobot.DiffRobot(1, 1, 1, 1, odomPublisher, posePublisher, lidar, 'formicarium/ant.png')
```

Comment:
Why do you use the ant.png for your test while the code is written with the robot.png?
Should the test not check the robot instead of the ant? Or is the code wrong and you want to use the ant.png?